packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Inbound/XCA ITI-39 Inbound Processor/destinationConnector-AWS Lambda Interface-transformer-step-0-Set Lambda params.js

```diff
@@ -1,7 +1,6 @@
 // Set the parameters required for the AWS Lambda
-channelMap.put('URL', configurationMap.get('INBOUND_DOCUMENT_RETRIEVAL_URL'));
-
-channelMap.put('DEST_ID', 'd' + connectorMessage.getDestinationIdMap().get('AWS Lambda Interface'));
+channelMap.put("URL", Config.getInboundDocumentRetrievalURL());
+channelMap.put("DEST_ID", "d" + connectorMessage.getDestinationIdMap().get("AWS Lambda Interface"));
 
 // Message ID required for Attachment
-channelMap.put('XCAMESSAGEID', java.lang.Long(connectorMessage.getMessageId()));
\ No newline at end of file
+channelMap.put("XCAMESSAGEID", java.lang.Long(connectorMessage.getMessageId()));
```

Comment:
Should we stick to one format here?
`XCAMESSAGEID` vs `DEST_ID`

Reply:
This is just formatting, replacing `'` by `"`, so I'd rather leave it like this since I'm not touching the logic, just replacing fixed variables by env vars.